packages/web-components/fast-ssr/src/element-renderer/fast-element-renderer.ts

```diff
@@ -175,8 +175,18 @@ function* renderAttributesSync(this: FASTElementRenderer): IterableIterator<stri
         ) {
             if (value === "" || value === undefined || value === null) {
                 yield ` ${name}`;
-            } else {
+            } else if (typeof value === "string") {
                 yield ` ${name}="${escapeHtml(value)}"`;
+            } else if (typeof value === "boolean") {
+                if (name.startsWith("aria-")) {
+                    yield ` ${name}="${(value as any).toString()}"`;
```

Comment:
Oops, I don't think we want any special casing for aria here. If the value is a boolean type, this should always yield `value.toString()`

Reply:
Discussed offline, will remove "aria-" related code. Whoever wants boolean attribute, need to follow "?attr-name" style from this doc: https://www.fast.design/docs/fast-element/declaring-templates/#attributes, it's handled within template renderer but not fast-element-renderer: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-ssr/src/template-renderer/template-renderer.ts#L371